inc/bdx86_core.h

```diff
@@ -347,15 +347,13 @@ typedef ND_UINT32 ND_REG_SIZE;
 // Misc macros.
 //
 
-// Sign extend 8 bit to 64 bit.
-#define ND_SIGN_EX_8(x)             (((x) & 0x00000080) ? (0xFFFFFFFFFFFFFF00 | (x)) : ((x) & 0xFF))
```

Comment:
Please don't remove existing macros. Even though they may be unused in bddisasm, there may be integrators out there depending on the.

Reply:
Done